memory-bank/systemPatterns.md

```
@@ -0,0 +1,254 @@
+  isTamed: boolean;
+}
+```
+
+**Level Scaling**:
+```
+levelMultiplier = 1 + (level - 1) * 0.1
+scaledStat = floor(baseStat * levelMultiplier)
+```
+
+**Biome-Element Mapping**:
+| Biome | Primary Elements |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)